Intermittent test failure: log-upgrade

https://app.circleci.com/pipelines/github/alxndrsn/odk-central-backend/3339/workflows/67b388e4-b306-4f5c-b74b-83c519472c5f/jobs/3196

```

  1) task: log-upgrade
       should log upgrade if client or server changes:

      AssertionError: expected Object { client: 'cf1', server: 'cb1', version: 'v1' } to equal Object { version: 'v1', server: 'cb2', client: 'cf1' } (at server, A has 'cb1' and B has 'cb2')
      + expected - actual

       {
         "client": "cf1"
      -  "server": "cb1"
      +  "server": "cb2"
         "version": "v1"
       }
      
      at Assertion.fail (node_modules/should/cjs/should.js:275:17)
      at Assertion.value (node_modules/should/cjs/should.js:356:19)
      at Context.<anonymous> (test/integration/task/log-upgrade.js:26:26)
      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
```

--- a/lib/model/query/audits.js
+++ b/lib/model/query/audits.js
@@ -64,7 +64,7 @@ const actionCondition = (action) => {
 
 // used entirely by tests only:
 const getLatestByAction = (action) => ({ maybeOne }) =>
-  maybeOne(sql`select * from audits where action=${action} order by "loggedAt", id desc limit 1`)
+  maybeOne(sql`select * from audits where action=${action} order by "loggedAt" desc, id desc limit 1`)
     .then(map(construct(Audit)));
 
 

